docs/src/dataDisplay/tag/tag.jsx

```diff
@@ -0,0 +1,282 @@
+                <Tag color="success" inverse={true}>Success</Tag>
+                <Tag color="primary" inverse={true}>Primary</Tag>
+                <Tag color="transparent" inverse={true}>Transparent</Tag>
+                <Tag color="warning" inverse={true}>Warning</Tag>
+            </div>
+        );
+    }
+
+}`;
+
+export const ON_CLICK_SAMPLE = `import { Tag } from 'react-cm-ui';
+import React from 'react';
+
+export default class OnClickSample extends React.Component {
+
+    render() {
+        return (
+            <Tag onClick={this._onClick.bind(this)}>Click Me</Tag>
+        );
+    }
+
+    _onClick() {
+        console.log('Label has been clicked!');
+    }
+
+}`;
+
+export const ON_CLEAR_CLICK_SAMPLE = `import { Tag } from 'react-cm-ui';
+import React from 'react';
+
+export default class OnClearClickSample extends React.Component {
+
+    render() {
+        return (
+            <Tag color="alert" onClearClick={this._onClearClick.bind(this)}>Alert</Tag>
+            <Tag color="highlight" onClearClick={this._onClearClick.bind(this)}>Highlight</Tag>
+            <Tag color="success" onClearClick={this._onClearClick.bind(this)}>Success</Tag>
+            <Tag color="primary" onClearClick={this._onClearClick.bind(this)}>Primary</Tag>
+            <Tag color="transparent" onClearClick={this._onClearClick.bind(this)}>Transparent</Tag>
+            <Tag color="warning" onClearClick={this._onClearClick.bind(this)}>Warning</Tag>
+        );
+    }
+
+    _onClearClick() {
+        console.log('Label removed.');
+    }
+
+}`;
+
+function DocsTag(props) {
+    const {
+        location: {
+            pathname,
+        },
+    } = props;
+
+    const {
+        description,
+        displayName,
+    } = rootDoc;
+
+    const onClick = useCallback(() => {
+        // eslint-disable-next-line no-console
+        console.log('Label has been clicked!');
+    }, []);
+
+    const onClearClick = useCallback(() => {
+        // eslint-disable-next-line no-console
+        console.log('Label removed.');
+    }, []);
+
+    return (
+        <Main page={camelCase(displayName)}>
+            <Main.Content>
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
+                    >
+                        {description}
+                    </Typography>
+                </MarkdownContainer>
+
+                {/* Label */}
+                <Header size="large" style={{ marginTop: '55px' }} sub>
+                    Label
+                    <Header.Subheader>
+                        A standard label that can be attached to something, giving information about it.
+                    </Header.Subheader>
+                </Header>
+
+                <Tag>Label</Tag>
+
+                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
+                    {LABEL_SAMPLE}
+                </Highlighter>
```

Comment:
Deliberately no attempt to update these doc pages and move older schtuff to "Dev Sandbox" and such?

Reply:
Correct

Reply:
Right-o.  Carry on!